src/main/java/appeng/core/features/registries/P2PTunnelRegistry.java

```diff
@@ -174,15 +164,15 @@ public void configure()
 		 * attune based on the ItemStack's modId
 		 */
 
-		this.addNewAttunement( "thermaldynamics", TunnelType.RF_POWER );
-		this.addNewAttunement( "thermalexpansion", TunnelType.RF_POWER );
-		this.addNewAttunement( "thermalfoundation", TunnelType.RF_POWER );
+		this.addNewAttunement( "thermaldynamics", TunnelType.FE_POWER );
+		this.addNewAttunement( "thermalexpansion", TunnelType.FE_POWER );
+		this.addNewAttunement( "thermalfoundation", TunnelType.FE_POWER );
 		// TODO: Remove when confirmed that the official 1.12 version of EnderIO will support FE.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)